client/src/components/Button/Button.css

```diff
@@ -0,0 +1,61 @@
+@import url("https://fonts.googleapis.com/css?family=Reem+Kufi&display=swap");
+
+.Button {
+  width: 200px;
+  height: 50px;
+  background: #fafafa;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 5px;
+  font-family: "Reem Kufi", sans-serif;
+  font-size: 24px;
+  line-height: 36px;
+  text-align: center;
+  color: #252525;
+  -webkit-transition-duration: 0.3s;
+  transition-duration: 0.3s;
+  -webkit-transition-property: background-color;
+  transition-property: background-color;
+}
+.Button:hover {
+  background-color: #fcf0ee;
+}
+.btn-large {
```

Comment:
If the base class is going to be `button`, then we should be consistent and use that throughout. Can you change these to `button-large`, `button-small`, etc?